tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/deid/fitbit_pid_rid_map_test.py

```diff
@@ -37,12 +37,18 @@ def setUpClass(cls):
         })
         cls.fq_deid_map_table = f'{project_id}.{mapping_dataset_id}.{mapping_table_id}'
 
-        cls.rule_instance = pr.FitbitPIDtoRID(project_id, cls.dataset_id,
-                                              cls.sandbox_id,
-                                              mapping_dataset_id,
-                                              mapping_table_id)
-
-        cls.fq_sandbox_table_names = []
+        cls.rule_instance = pr.FitbitPIDtoRID(
+            project_id,
+            cls.dataset_id,
+            cls.sandbox_id,
+            table_namer=None,
+            mapping_dataset_id=mapping_dataset_id,
+            mapping_table_id=mapping_table_id)
```

Comment:
I would probably leave this one as it is written above.
```suggestion
        cls.rule_instance = pr.FitbitPIDtoRID(project_id, cls.dataset_id,
                                              cls.sandbox_id,
                                              mapping_dataset_id,
                                              mapping_table_id)
```
This identifies that `mapping_dataset_id` and `mapping_table_id` are required.  By not providing `table_namer` we are defaulting to the `None` value anyway.  What do you think?

Reply:
You are right - I would like to keep it simple, so let me make this change as well. Thanks :)